src/System.Private.Xml/src/System/Xml/Xsl/Runtime/XmlSortKey.cs

```diff
@@ -219,6 +219,79 @@ public override int CompareTo(object obj)
     }
 
     /// <summary>
+    /// Sort key for xs:string values.  Strings are sorted according to a byte-wise sort key calculated by caller.
+    /// </summary>
+    internal class XmlStringSortKey : XmlSortKey
```

Comment:
@justinvp I'm happy to make that change, but since there are other `Xml*SortKey` classes here which are not sealed and it could be nice to have consistency among them, may I ask why you think we should make this one `sealed`?